lib/confluent_schema/server.ex

```diff
@@ -55,10 +64,4 @@ defmodule ConfluentSchema.Server do
         end
```

Comment:
If I understood this correctly, if debug is set to false in the lines above, then this branch will return `nil`. Is this silent failure intended behavior for callers of the `cache` function ?

Reply:
Yes, it is the behaviour, since `cache` is a function focused on external side-effects, we are not interested in its return